docker-compose/docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
         condition: service_started
 
   winearth-api-server:
-    image: ghcr.io/windows-on-earth/winearth-api:0.1.5
```

Comment:
I don't understand why this is happening, I don't think you have the latest version of the code before submitting the pull request.